### ISSUE TITLE
BUG - Sev 3 - AndroidSamsung - Screenreader doesn't read > or <

### ISSUE THREAD

**Issue by dumathane** [department-of-veterans-affairs/va-mobile-app#11182]
## What happened?

Samsung screenreader does not read < or > appropriately which could cause issues interpreting medical data.
## Specs:

- Device: Samsung
- OS: Android
- User Account: Any
- Accessibility State: Screenreader on, advanced symbol settings off.

## Steps to Reproduce

On a samsung device go attempt to read > and < in any context.

## Desired behavior

These should be read.

## Acceptance Criteria

Fixing the bug should fix any place in the app where > and < are read.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High
- Frequency: Low 

- 3 - LOW

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by BeccaTupaj** [comment]
Refinement on 7/9: Team determined that this is not a bug in the app. Possible next step is to raise to Rx Team that this issue exists for some users and ask if there is possibility of them changing format of prescriptions.
